spec/knackhq/client_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'spec_helper'
+
+describe Knackhq::Client do
+  it 'has a version number' do
+    expect(Knackhq::Client::VERSION).not_to be nil
+  end
+
+  let(:client) { knack_client }
+  let(:base_uri) { 'http://example.com' }
+  let(:x_knack_application_id) { '00000000' }
+  let(:x_knack_rest_api_key) { '99999999' }
+
+  let(:knack_client) do
+    Knackhq::Client.new(base_uri,
+                        x_knack_application_id,
+                        x_knack_rest_api_key)
+  end
+
+  describe '#new' do
+    it { expect(knack_client).to be_a Knackhq::Client }
+  end
+
+  describe '#request.get' do
```

Comment:
Ill look at the documentation with Blanket and see what I can find. What kind of tests would be beneficial that will not test Blanket as a gem VS just testing to see if we get a response?